_docs_explore/22-security.md

```diff
@@ -41,9 +41,45 @@ However a malware could still access the data within the Enmeshed Runtime, meani
 The Connector should be part of the organizations network, otherwise there would be privacy issues with unencrypted payload or third-parties being able to impersonate the organization. It acts a central integration point within the organization and thus can be easily kept up-to-date.
 In our opinion, this is a far better approach than to upgrade any business system with encryption and communication capabilities.
 
-# App Security
+# Security Recommendations for Apps and End-Users
 
-The App is only communicating with its corresponding Backbone.
+Security is a crucial aspect of any application, and as an end-user, there are various measures that you can take to ensure that your device and the app are safe from potential threats. In this chapter, we will highlight some of the most important security recommendations for both the app and end-users.
+
+## Enable PIN/Password Authentication of Device
+
+The first line of defense for your device is a strong password or PIN. It is essential to enable this feature on your device to prevent unauthorized access. A strong password or PIN should be at least six characters long, contain a mix of upper and lowercase letters, numbers, and symbols. Additionally, you should avoid using easily guessable passwords, such as birthdays or names.
```

Comment:
> should be at least six characters long, contain a mix of upper and lowercase letters, numbers, and symbols

Hier kommt es stark darauf an, ob du jetzt von einem Passwort oder einer PIN redest. Ich weiß nicht ob wir das vereinheitlichen sollten.
Beispiele:
* Bei einer PIN kannst du ja oftmals keine Buchstaben angeben, daher ist "upper and lowercase letters" vielleicht irreführend. 
* Ein Passwort sollte mehr als nur 6 Zeichen lang sein

Reply:
ein PIN in Windows kann z.B. auch Zeichen haben ..

Reply:
Deshalb auch nur "oftmals". 
Aber wenn das ein Smartphone-Nutzer liest, ist dieser wahrscheinlich trotzdem verwirrt.

Ist aber auch nur ein Vorschlag. Ich beharre da nicht drauf :)